src/commands/force/user/password/generate.ts

```diff
@@ -26,6 +27,14 @@ export class UserPasswordGenerateCommand extends SfdxCommand {
       char: 'o',
       description: messages.getMessage('flags.onBehalfOf'),
     }),
+    length: flags.number({
```

Comment:
flags can have a min/max.  At least a min would be good here (there may not be a max)

Reply:
and add the `default` property for the flag